docs/source/release/v6.8.0/Diffraction/Powder/New_features/35783.rst

```diff
@@ -0,0 +1 @@
+* Update HRPD definition to reflect modifications to low-angle bank modules for runs from 01-MAR-23 onwards
```

Comment:
Can you use a `-` instead of `*` for the bullet point? Using a `*` can cause issues further down the line for the Release Editors

```suggestion
- Update HRPD definition to reflect modifications to low-angle bank modules for runs from 01-MAR-23 onwards
```